statusbarwidget.h

```diff
@@ -0,0 +1,28 @@
+#ifndef STATUSBAR_H
+#define STATUSBAR_H
+
+#include "iaggragateddataprovider.h"
+
+#include <QWidget>
+#include <QLabel>
+#include <QStatusBar>
+#include <QHBoxLayout>
+
+class StatusBarWidget : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit StatusBarWidget(QWidget *parent = 0);
+    ~StatusBarWidget(){}
+    void showTemporaryStatus(const QString &message);
+    void changePopulationStatisticsMessage(IAggragatedDataProvider *data);
+private:
+    QLabel m_minPopulation;
```

Comment:
Qt controls are typically not created on stack, create them using new operator and pass StatusBarWidget as parent